sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -58,6 +58,15 @@ public DataFeedLiveTests(bool isAsync) : base(isAsync)
         {
         }
 
+        private DateTimeOffset IngestionStartsOnForCustom
+        {
+            get
+            {
+                DateTimeOffset startsOn = Recording.UtcNow.Subtract(TimeSpan.FromDays(2));
+                return new DateTimeOffset(startsOn.Year, startsOn.Month, startsOn.Day, startsOn.Hour, startsOn.Minute, 0, TimeSpan.Zero);
```

Comment:
The service requires the date specified to be rounded to seconds for the type of granularity we're using (custom), so we're transforming the `DateTimeOffset` here.